app/src/main/java/com/example/criminalintent/presentation/crime_detail/CrimeFragment.kt

```diff
@@ -36,14 +44,20 @@ class CrimeFragment : Fragment(), FragmentResultListener {
     private lateinit var solvedCheckBox: CheckBox
     private lateinit var reportButton: Button
     private lateinit var suspectButton: Button
+    private lateinit var photoButton: ImageButton
+    private lateinit var photoView: ImageView
 
-    private val crimeDetailViewModel: CrimeDetailViewModel by lazy {
+    private val viewModel: CrimeDetailViewModel by lazy {
         ViewModelProvider(this).get(CrimeDetailViewModel::class.java)
     }
 
-    private var resultLauncher =
+    private var resultLauncherSuspect =
```

Comment:
can `updatePhotoView` and `resultLauncherPhoto` be `val`s?

Reply:
did you run detekt and ktlintCheck?